sdk/servicebus/Azure.Messaging.ServiceBus/src/Plugins/ServiceBusPlugin.cs

```diff
@@ -29,6 +33,58 @@ public abstract class ServiceBusPlugin
         public virtual ValueTask AfterMessageReceiveAsync(ServiceBusReceivedMessage message) =>
             default;
 
+        internal virtual ValueTask ProcessSendAsync(ServiceBusMessage message, ReadOnlyMemory<ServiceBusPlugin> pipeline)
+        {
+            string pluginType = GetType().Name;
```

Comment:
This isn't likely to change.   Should we consider a `readonly` member to that we can reuse?   Or, maybe an abstract protected property for `Name` to let implementers offer a friendly name?

Reply:
> Or, maybe an abstract protected property for Name to let implementers offer a friendly name?

We got read of it in the previous review I would avoid adding it until there's a strong ask from customers.

Reply:
Yeah I can at least set this in the base ctor.